demos/rerank/README.md

```diff
@@ -16,142 +16,85 @@ It will create an image called `openvino/model_server:latest`.
 > **Note** Python 3.9 or higher is needed for that step
 Here, the original Pytorch LLM model and the tokenizer will be converted to IR format and optionally quantized.
 That ensures faster initialization time, better performance and lower memory consumption.
-LLM engine parameters will be defined inside the `graph.pbtxt` file.
 
 Install python dependencies for the conversion script:
 ```bash
-export PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
-pip3 install optimum-intel@git+https://github.com/huggingface/optimum-intel.git  openvino-tokenizers[transformers]==2024.4.* openvino==2024.4.* nncf>=2.11.0 sentence_transformers==3.1.1 openai "transformers<4.45"
+pip3 install -r https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/export-models-script/demos/common/export_models/requirements.txt
```

Comment:
Branch will be deleted

Reply:
after branch export-models-script is merged, it will be updated to main